app/scripts/build-and-push-montagu

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure what this script is for. Why does packit contain a script that builds montagu?

Reply:
Relates to this bit of the PR description:
"new gha workflow, `build_montagu_packit_front_end`. and related script `build-and-push-montagu` which builds a front end image which bakes in the front end and back end routes we expect to use for packit under the top level montagu domain (we may change this to use a subdomain in a future ticket). Related Dockerfile changes to pick up these build args."

This is an unfortunate consequence of assuming that we'd be using the nix deployer from here on out, which builds the front end on deploy rather than using a pre-built docker image..

Reply:
It's run as part of CI, in the `build_montagu_packit_front_end` workflow, which just builds and pushes a slightly different docker image with the required env vars baked in, just for use by montagu. Other deployments don't need to do that because they use the nix deployment, which builds the front end from scratch at the point of deployment. 

It's all a bit unfortunate. Let's chat about it on Monday if still not clear.

Reply:
Does this script build the whole of montagu, or just a montagu-flavoured version of packit?

Reply:
The latter! I will rename..